cometr/local_metrics/AE.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import os
+from os.path import dirname, abspath
+
+import h5py
+import numpy as np
+import torch
+from beartype import beartype
+
+
+from cometr.Metric import Metric
+
+
+class AE(Metric):
```

Comment:
perhaps consider a more descriptive class name to allow the class to 'self document' when used.

for example, using this class currently, would look something like this

```
from cometr.local_metrics.AE import AE
object = AE(metric)
```

whereas if you name the classes and functions clearly, the user can abbreviate them when they want to shorten but when just glancing at code they right, you can quickly identify what its doing. example

```
from cometr.local_metrics.AbsoluteError import AbsoluteErrorCalculation
object = AbsoluteErrorCalculation(metric)
```

This is all preference and hard to show in an example but imagine you have 300 lines of code between your import and object. If the object is running AE, you have to go search for what that means whereas with my suggestion its clear what that object does.

As i said, this is preference but to make the package clearer, this might be nice

Reply:
Thanks. I have updated it to be more descriptive.